### PR TITLE
New temp

### PR DESCRIPTION
**Pull request by tarabhatia**
There are some changes.

### CODE REVIEW

File: app/src/main/java/com/example/fixit/MainActivity.java [hunk 1]
```diff
@@ -42,15 +42,15 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View v) {
                 SignUp();
-//                Intent intent = new Intent(MainActivity.this, UserActivity.class);
-//                startActivity(intent);
-//                finish();
+                Intent intent = new Intent(MainActivity.this, UserActivity.class);
+                startActivity(intent);
+                finish();
```
Comment (bearcub-misha) [inline app/src/main/java/com/example/fixit/MainActivity.java:47]: Why?

**Review by bearcub-misha — CHANGES_REQUESTED** [review]
COMMENT

File: app/src/main/java/com/example/fixit/MainActivity.java (hunk 1 above)
Comment (tarabhatia) [reply app/src/main/java/com/example/fixit/MainActivity.java:47]: reply

**Comment by bearcub-misha** [comment]
Looks good to me.

**Review by bearcub-misha — APPROVED** [review]

**Pull request merged**